_protected/app/system/modules/mail/controllers/MainController.php

```diff
@@ -251,10 +260,10 @@ public function trash()
             );
 
             if (empty($oMail)) {
-                $this->sTitle = t('Sorry!');
+                $this->sTitle = t('Not Found!');
```

Comment:
Much better message! Thanks!

Reply:
I just changed a few of the messages like this as we are saying sorry for no reason :) 

Also named the page titles as messages rather than mail as mail is what a postman brings haha. and email is in no way related to pm's :)